src/components/ec-metroline-item/ec-metroline-item.vue

```diff
@@ -0,0 +1,203 @@
+<template>
+  <div
+    :data-test="$attrs['data-test'] ? `${$attrs['data-test']} ec-metroline-item` : 'ec-metroline-item'"
+    class="ec-metroline-item"
+    :class="{
+      'ec-metroline-item--active': isActive || ( isCompleted && isLast),
```

Comment:
We do have `--is-last` modifier, this one should be then `--is-active`.

Reply:
?